packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbarContent.tsx

```diff
@@ -12,17 +13,107 @@ export interface DataToolbarContentProps extends React.HTMLProps<HTMLDivElement>
   breakpointMods?: DataToolbarBreakpointMod[];
   /** Content to be rendered as children of the content row */
   children?: React.ReactNode;
+  /** Flag indicating if a Data toolbar toggle group's expandable content is expanded */
+  isExpanded?: boolean;
```

Comment:
Is this for the toggle group or content? The description, "toggle group's expandable content is expanded" sounds confusing to me.

Considering there is a `toggleIsExpanded` prop below, perhaps something more like `isContentExpanded` would help to clarify the use case?

Reply:
so the toggle group has an associated expandable content that is what gets expanded or collapsed when the toggle icon in the toggle group is clicked. It is confusing. I feel passing an `isExpanded` flag as a prop to the `DataToolbarToggleGroup` is the clearest way to state it. Do you have another idea?

Reply:
`isConsumerManagedToggleGroup` is a state field calculated based on whether or not the consumer provided `isExpanded` and the `toggleIsExpanded` handler, not a passed prop. But I will rename it to `isToggleManaged` and make the comments clearer?

Reply:
gotcha, ok